Starting with check 0.13.0, check incorrectly checks for the tex instead
of the makeinfo command for building documentation. 

See:

	* https://github.com/libcheck/check/issues/206
	* https://github.com/libcheck/check/pull/212

Without this change documentation files (the info page and example code)
which were previously installed are no longer installed.

diff -upr check-0.13.0.orig/configure.ac check-0.13.0/configure.ac
--- check-0.13.0.orig/configure.ac	2019-10-25 17:17:54.361921908 +0200
+++ check-0.13.0/configure.ac	2019-10-25 17:18:13.101931933 +0200
@@ -182,7 +182,7 @@ AC_CHECK_PROGS(LCOV, lcov, false)
 AC_CHECK_PROGS(GENHTML, genhtml, false)
 
 if test "xtrue" = x"$enable_build_docs"; then
-    AC_CHECK_PROGS(TEX, tex, false)
+    AC_CHECK_PROGS(TEX, makeinfo, false)
     if test "$TEX" = "false"; then
         # Make it [somewhat] clear to maintainers that tex is missing. Not an error
         # though because 'make install' (which users need) does not build the docs
